rasa/shared/core/domain.py

```diff
@@ -641,6 +641,14 @@ def raise_action_not_found_exception(self, action_name) -> NoReturn:
         )
 
     def random_template_for(self, utter_action: Text) -> Optional[Dict[Text, Any]]:
+        """Great docstring but no dots at the end
```

Comment:
it says "First line should end with a period, question mark, or exclamation point", does it support multi-line though? Would:
```suggestion
        """Great docstring but no dots
        at the end
```
Fail? What if there is the dot only on the second line?

Reply:
The first line should actually never be multiline as stated [here](https://google.github.io/styleguide/pyguide.html#382-modules)

Reply:
I didn't know about this recommendation. Not sure everyone is aware

Reply:
I think it makes a lot of sense to keep the summary a summary + that's a habit which should be quick to adapt (in the end it forces you to write less 😂 )